### PR TITLE
Open close menu

### PR DESCRIPTION
**Pull request by thecodechaser**
This branch includes the following features :
Added javascript, DOM for mobile menu
-When click on the hamburger button, the menu open
-When click on the close button, the menu close 
-When click on any menu item, menu close and take you to clicked section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,12 +38,33 @@
...
           </div>
-          <div class="menu-button-burger"></div>
+
+          <div class="mobile-nav-container">
+            <div class="btn-container-open">
+              <button class="btn-open" type="button">
+                <img
+                  class="btn-open-img"
+                  src="images/hamburger.png"
+                  alt="hamburger menu"
```
Comment (rotimiazeez) [inline index.html:48]: - [ ] Kindly make sure the hamburger button and menu follow the design from the chosen template in Figma (e.g., font, colours, images, tests, margins).👌
- Your design
screenshot (15)
- Figma design
screenshot (16)

File: css/stylesheet.css [hunk 2]
```diff
@@ -34,30 +34,68 @@ header {
...
 
-.menu-button-burger {
-  width: 30px;
-  height: 4px;
-  background: #6070ff;
+.btn-open {
+  margin-top: 20px;
+  border: none;
+  cursor: pointer;
+}
```
Comment (rotimiazeez) [inline css/stylesheet.css:44]: - [ ] Kindly make sure your design matches your preferred design😉.
- Figma design
screenshot (17)
- Your design
screenshot (18)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team 👋, 

### Good job so far👏👏👏


### To Highlight 👏
- No linter errors✔
- Descriptive PR message👏
- Good README 💯

There are some issues that you still need to work on to go to the next project but you are almost there!👍💯.
### Great job so far 👍 just a few changes and you are on to the next level😎💯

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @thecodechaser, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**